torchreid/utils/torchtools.py

```diff
@@ -307,74 +437,6 @@ def _print_loading_weights_inconsistencies(discarded_layers, unmatched_layers):
         )
 
 
-def load_pretrained_weights(model, file_path='', pretrained_dict=None):
```

Comment:
Let's avoid changing the functions declaration order to produce a compact git diff